src/CoreBundle/Resources/public/javascript/tableEditor.js

```diff
@@ -868,6 +868,14 @@ CHAMELEON.CORE.MTTableEditor.initSelectBoxes = function () {
             });
         });
     });
+
+    $('.select2.select2-container').on('keydown', function(e) {
+        if (40 === e.keyCode) {
+
+            var $selectBox = $(e.target).parent().parent().siblings('select')[0];
```

Comment:
This lookup seems a bit fragile. Can we jump to a named parent element and then down to the select element?

Also, can we avoid dereferencing the first element here, as we wrap it in a jQuery element in the next line anyway?

Reply:
closes() with a selector was the solution here.